frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -90,18 +90,20 @@ export const TicketDetailsCard = () => {
 	};
 
 	useEffect(() => {
-		if (!templateAlreadyFetched(template)) {
-			TicketsActionsDispatchers.fetchTemplate(
-				teamspace,
-				project,
-				containerOrFederation,
-				template._id,
-				isFederation,
-			);
+		if (ticket?._id) {
```

Comment:
To make it more concise, can you make this if a return statement if the condition is false?
Like this
```ts
if (!ticket?.id) return;

if (!templateAlreadyFetched(template) {
   ...
```